src/Models/Profile.js

```diff
@@ -19,4 +23,59 @@ export default class Profile {
     get email() {
         return this._email;
     }
+
+    get position() {
+        return this._position;
+    }
+
+    get companyName() {
+        return this._companyName;
+    }
+
+    get phoneNumber() {
+        return this._phoneNumber;
+    }
+
+    set displayName(displayName) {
+        this._currentUser.updateProfile({
+            displayName: displayName,
+        }).then(() => {
+            this._displayName = this._currentUser.displayName;
+        }).catch((error) => {
+            console.error(error.message);
+        });
+    }
+
+    set photoURL(photoURL) {
+        this._currentUser.updateProfile({
+            photoURL: photoURL,
+        }).then(() => {
+            this.photoURL = this._currentUser.photoURL;
+        }).catch((error) => {
+            console.error(error.message);
+        });
+    }
+
+    set email(email) {
+        this._currentUser.updateEmail(email).then(() => {
+            this._email = this._currentUser.email;
+        }).catch((error) => {
+            console.error(error.message);
+        });
+    }
+
+    set phoneNumber(phoneNumber) {
+        // TODO: Save to Firebase - no option to save it with updateProfile method...
```

Comment:
nie wiem czy nie lepiej zrobic save() na calym profilu a nie na poszczegolnych polach, ale to do dyskusji